src/components/Table/hoc/withTableSettings/TableColumnSetup/TableColumnSetup.tsx

```diff
@@ -24,7 +24,10 @@ import i18n from './i18n';
 
 import './TableColumnSetup.scss';
 
-function identity<T>(value: T): T {
+function identity(value: Item): ItemRenderProps {
+    delete value.isSelected;
+    delete value.isRequired;
```

Comment:
Mutating parameter is bad, can we resolve this another way?

Reply:
Sure, 7f624b2a1726f0f2d24d5f3d290c489d62537fc3